app/screens/home/channel_list/index.ts

```diff
@@ -21,11 +21,22 @@ const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
         switchMap((lcs) => (lcs ? of$(lcs.IsLicensed === 'true') : of$(false))),
     );
 
+    const teamsCount = queryMyTeams(database).observeCount(false);
+
     return {
         isCRTEnabled: observeIsCRTEnabled(database),
-        teamsCount: queryMyTeams(database).observeCount(false),
-        channelsCount: observeCurrentTeamId(database).pipe(
+        hasTeams: teamsCount.pipe(
```

Comment:
These changes help with minor unneeded re-renders on team switch due to the number of channels changing, when our real interest is whether we have more than one or not.